lib/openstates/models/bill.rb

```diff
@@ -32,7 +32,7 @@ def api_method
       end
 
       def id_key
-        :bill_id
+        :id
```

Comment:
It's only used for the ```find``` method, and Bill#find should use ```id``` instead of ```bill_id```. I changed so the nomenclature makes more sense.